MuxPhp/Models/IncidentNotification.php

```diff
@@ -0,0 +1,333 @@
+<?php
+
+/**
+ * Mux PHP - Copyright 2019 Mux Inc.
+ * NOTE: This file is auto generated. Do not edit this file manually.
+ */
+
+namespace MuxPhp\Models;
+
+use \ArrayAccess;
+use \MuxPhp\ObjectSerializer;
+
+/**
+ * IncidentNotification Class Doc Comment
+ *
+ * @category Class
+ * @package  MuxPhp
+ */
+class IncidentNotification implements ModelInterface, ArrayAccess
+{
+    const DISCRIMINATOR = null;
+
+    /**
+     * The original name of the model.
+     *
+     * @var string
+     */
+    protected static $openAPIModelName = 'IncidentNotification';
+
+    /**
+     * Array of property to type mappings. Used for (de)serialization
+     *
+     * @var string[]
+     */
+    protected static $openAPITypes = [
+        'queued_at' => 'string',
+        'id' => 'double',
```

Comment:
:( we made this a number didn't we... still not sure why `double` here, but seems like just how it's done.